chain/client/src/info.rs

```diff
@@ -48,6 +48,8 @@ pub struct InfoHelper {
     validator_signer: Option<Arc<dyn ValidatorSigner>>,
     /// Telemetry actor.
     telemetry_actor: Addr<TelemetryActor>,
+    /// Log coloring enabled
+    log_color: bool,
```

Comment:
Considering the whole structure, I find this name to be not very descriptive. Consider something along these lines: `should_use_color_for_logging` (I am not totally sold on it as it is quite long, but at least it is immediately obvious how one may want to use it)

Reply:
colored_logging_enabled maybe would be better?

Reply:
> colored_logging_enabled maybe would be better?

Yeah let's do that